cliv2/internal/proxy/proxy.go

```diff
@@ -71,8 +71,8 @@ func InitCA(config configuration.Configuration, cliVersion string, logger *zerol
 		return nil, err
 	}
 
-	tmpDirectory := utils.GetTemporaryDirectory(cacheDirectory, cliVersion)
-	err = utils.CreateAllDirectories(cacheDirectory, cliVersion)
+	tmpDirectory := pkg_utils.GetTemporaryDirectory(cacheDirectory, cliVersion)
```

Comment:
Issue: with the new gaf version, the temp directory is a configuration value. Instead of using this helper function directly, we should be using the configuration now, otherwise the behaviour wouldn't be consistent.

```suggestion
	return config.GetString(configuration.TEMP_DIR_PATH)
```